src/main/scala/beam/agentsim/agents/choice/mode/ModeChoiceLCCM.scala

```diff
@@ -136,7 +136,7 @@ class ModeChoiceLCCM(val beamServices: BeamServices, val lccm: LatentClassChoice
       ModeChoiceData(altAndIdx._1.tripClassifier, tourType, vehicleTime, walkTime, waitTime, bikeTime, totalCost.toDouble, altAndIdx._2)
     }
 
-    val groupedByMode = modeChoiceAlternatives.sortBy(_.mode.value).groupBy(_.mode)
+    val groupedByMode: Map[BeamMode, Seq[ModeChoiceData]] = modeChoiceAlternatives.groupBy(_.mode)
```

Comment:
I believe that the sort will indeed be lost in the long term, but was this meant to be an attempt to sort the values within the grouping? I don't see any problem with the optimization, and am just curious if the original intent was lost and should be fixed instead of optimized?

Reply:
It's hard to say about original intent, maybe @colinsheppard can help here.